src/src/Helpers/ESPEasy_checks.cpp

```diff
@@ -178,7 +178,7 @@ void run_compiletime_checks() {
   static_assert(198u == offsetof(SettingsStruct, TaskDeviceNumber), "NOTIFICATION_MAX has changed?");
 
   // All settings related to N_TASKS
-  static_assert((200 + TASKS_MAX) == offsetof(SettingsStruct, OLD_TaskDeviceID), ""); // 32-bit alignment, so offset of 2 bytes.
+  static_assert((236 + TASKS_MAX) == offsetof(SettingsStruct, OLD_TaskDeviceID), ""); // 32-bit alignment, so offset of 2 bytes.
```

Comment:
Ehh this is something we really should look into.
Those `offsetof` checks really should never ever trigger or else existing settings will become unreadable.

Reply:
I have reused the lower elements of the `OLD_TaskDeviceID` array as storage for the new settings fields, so the start address of this variable has changed.